examples/ParRes/standalone/nstream.py

```diff
@@ -4,15 +4,15 @@
 import sys
 
 @pk.workunit
-def init_view(i: int, inp: pk.View1D[pk.double], init: float):
+def init_view(i, inp, init):
     inp[i] = init
 
 @pk.workunit
-def nstream( i: int, A: pk.View1D[pk.double], B: pk.View1D[pk.double], C: pk.View1D[pk.double], scalar: float):
+def nstream( i, A, B, C, scalar):
```

Comment:
in some files we use lower case letters for arguments and in some cases uppercase.  Let's use lowercase always until we have already convention otherwise (in which case we should update those that are lowercase to uppercase).

Reply:
Same as above, I believe this is copied from the original examples. It's fine to leave it IMO.